client/components/plans/plans-compare/index.jsx

```diff
@@ -123,7 +129,9 @@ const PlansCompare = React.createClass( {
 	},
 
 	isDataLoading() {
-		if ( ! this.props.features.get() ) {
+		const planFeatures = this.props.features.get();
+
+		if ( ! planFeatures || isEmpty( planFeatures ) ) {
```

Comment:
This is the culprit of the issue. `this.props.features.get()` returns an empty array if the store doesn't return any data. Therefore, this condition was never `true` and therefore the placeholders were not showing.